test/logging/example_tlogger_test.go

```diff
@@ -34,8 +34,7 @@ var (
 
 func init() { someStruct = testStruct{"hello", 42.0} }
 
-// godoc limitation; this would be named Test*(), of course
-
+// nolint:govet // godoc limitation; this would be named Test*(), of course
```

Comment:
Totally can get rid of the linter warning by naming `ExampleTLogger`, I am quite certain.

Reply:
Nope, it complains about the function not being niladic because Examples shouldn't take parameters at all.